templates/Program/program_description.html.twig

```diff
@@ -117,57 +56,54 @@
 <div class="row">
   <div class="col">
     {% if app.user and my_program %}
-      <div id="edit-credits-ui" class="d-none mb-5">
-        <div class="mt-3" id="credits-headline">
-          <span class="h2">{{ "credits"|trans({}, "catroweb") }} </span>
-          {% if app.user and my_program %}
+      <div id="edit-text-ui" class="d-none mb-5">
+        <div class="mt-3">
+          <span class="h2" id="edit-headline"></span>
             <span>
-              <i data-bs-toggle="tooltip" title="{{ 'programs.closeCreditsEditor'|trans({}, 'catroweb') }}"
-                 id="close-credits-editor-button" class="ml-3 material-icons catro-icon-button align-bottom">
+              <i data-bs-toggle="tooltip"
```

Comment:
By removing the title here, does that also remove the tooltip explanation?

Reply:
The title is added dynamically here, if that's what you're wondering

https://github.com/danielzhang130/Catroweb/blob/4a808836e64b2caf4155f03fb994f15f60adff3d/assets/js/components/ProjectTextEditor.js#L71